src/WheelsAndGoods.DataAccess.Migrations/Migrations/20220818165257_AddRefreshTokenTable.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using Microsoft.EntityFrameworkCore.Migrations;
+
+#nullable disable
+
+namespace WheelsAndGoods.DataAccess.Migrations.Migrations
+{
+    public partial class AddRefreshTokenTable : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.CreateTable(
+                name: "refresh_token",
```

Comment:
Wrong column name. Should be refresh_tokens. Looks like you are not added DbSet to Context.